components/io_expander/esp_io_expander_tca95xx_16bit/test_apps/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# The following lines of boilerplate have to be in your project's CMakeLists
+# in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.5)
+set(EXTRA_COMPONENT_DIRS "$ENV{IDF_PATH}/tools/unit-test-app/components")
```

Comment:
I don't see any function from `$ENV{IDF_PATH}/tools/unit-test-app/components/test_utils` in this test. Can it be removed (in all 3 test_apps)?